xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Well, no. I'm trying to be very cooperative as you might have expirienced already, but doing it this way is good practice. 

Regarding private: May I friendly advice you to take a look Scott Meyers "More Effective C++" chapter 5.2. 
Regarding friend: From the architectural point of view CPVRRecordings is a factory for CPVRRecording instances. Thus, it is just fine to make it a friend.